src/plugin.scss

```diff
@@ -82,40 +88,21 @@
 
 .vjs-errors-content-container {
     overflow: auto;
-    position: absolute;
+    position: relative;
     padding-bottom: 15px;
-    top: 75px;
+    top: -10px;
```

Comment:
Another thing to think about: do we actually need to make changes to the previous styles? One  thing that changed here it the close button, can we make it so that it doesn't affect the previous styles?